packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -96,6 +100,7 @@ const RekognitionAndTextract = {
   'eu-west-3': 'eu-west-1',
   'eu-central-1': latestTextractRegions >= 1 ? 'eu-central-1' : 'us-east-1',
   'eu-north-1': 'eu-central-1',
+  'eu-south-1': 'eu-central-1',
```

Comment:
Textract is not available in eu-south-1. Falling back to eu-central-1.

https://docs.aws.amazon.com/general/latest/gr/textract.html